nnvm/tests/python/frontend/tensorflow/test_forward.py

```diff
@@ -502,6 +502,43 @@ def test_forward_gather():
     _test_gather((4,3,5,6), (1,4), [[2,1,0,0]], 0, 'float32')
 
 
+#######################################################################
+# Split
+# ------
+
+def _test_split(in_shape, axis, num_split, dtype):
```

Comment:
Can you add another test case with split followed by concat ?

Reply:
@srkreddy1238 updated